desc/compute/_basis_vectors.py

```diff
@@ -688,6 +692,10 @@ def _n_rho(params, transforms, profiles, data, **kwargs):
     profiles=[],
     coordinates="rtz",
     data=["e_rho", "e_zeta", "|e_zeta x e_rho|"],
```

Comment:
So the e_rho and e_zeta dont seem to have parametrizations, is there a reason for that? And how would e_rho defined for the toroidal RZ surface, is it just zero?

Reply:
nevermind, I see it is in the surface.py, and the default parametrization is for Equilibrium